local-interchain/cmd/local-ic/new_chain.go

```diff
@@ -185,7 +185,3 @@ func getOrDefault[T any](output string, defaultVal T) T {
 
 	return any(text).(T)
 }
-
-func init() {
-	rootCmd.AddCommand(newChainCmd)
-}
```

Comment:
I avoid `init()` functions at all costs. Even though this is common in Viper and co-located with the subcommand file, I think it's an anti-pattern. It hides order of operations and hides functionality in the `main()` function. IOW, I want to look at `main()` and know exactly what's happening.